<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourWindow</class>
 <widget class="QDialog" name="FourWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>731</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">backgroung-color:none</string>
   </property>
   <property name="text">
    <string>To get involved with GetEncoureged, you can become the newest memeber of us !! </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Fill out the form below to apply:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your Major:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What would you like to tell your prospective mentee in a short presentation about yourself?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:#FDFEEB  ;
}</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>GetEncouraged</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FDFEEB</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FDFEEB</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FDFEEB</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textInput2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FDFEEB</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
